src/others/contexts/file-downloader.tsx

```diff
@@ -88,7 +88,7 @@ export const FileDownloaderContextProvider: FunctionComponent = ({ children }) =
   const downloadAsCSV = (aidRequests: AidRequest[]) => {
     const [ formattedAidRequests, uniqueCategories ] = formatAidRequests(aidRequests);
     const csvRows = [];
-    const csvHeaders = ["City", "Date"].concat(Array.from(uniqueCategories));
+    const csvHeaders = ["City", "Date"].concat(uniqueCategories);
```

Comment:
**question**

Will it work for multiple cities selected?

Reply:
Yeah, because cities are rows and categories are columns.